capstone/fitness/src/main/webapp/script.js

```diff
@@ -90,12 +99,18 @@ function createLoginTemplate(name, url, type) {
   return template;
 }
 
-// TODO: load in user information from LoginServlet.java
-// TODO: load in user's scheduled events from CalendarServlet.java
-// DONE: get current date.
+async function getUser(){
+  const loginResponse = await fetch('/login');
+  const loginInfo = await loginResponse.json();
+  
+  const userEmail = loginInfo.email;
+  usernameContainer = document.getElementById("user-message");
+  usernameContainer.innerHTML = "Hello " + userEmail + ".";
+}
+//TODO (@piercedw) :
+//Fetches JSON from server and displays exercises.
 async function getCalendarInfo(){
```

Comment:
this method seems to do the same thing as getData(). Do you need it?

Reply:
No. I think it's vestigial from old code. I will remove it.